src/main/resources/db/changelog/changes/12-add-cartitem-table.yaml

```diff
@@ -0,0 +1,36 @@
+databaseChangeLog:
+  - changeSet:
+      id: create cart_item-table
+      author: Yuriy Kononchuk
+      changes:
+        - createTable:
+            tableName: cart_item
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: shopping_cart_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
![image](https://github.com/yuriy-kononchuk/Online-Book-store-App-YK/assets/49767049/a6732d63-88d7-4292-8b27-985270247bff)
add empty line everywhere you see this symbol

Reply:
done